src/en/space-station-14/departments/science/proposals/scientific-research-proposal.md

```diff
@@ -0,0 +1,36 @@
+# Scientific research proposal
+
+| Designers     | Implemented | GitHub Links |
+|---------------|-------------|--------------|
+| chromiumboy   | :x: No      | TBD          |
+
+## Problem statement
+At present the science department faces several key gameplay issues when it comes to unlocking items through scientific research. These include (but are not limited to):
+-	A large number of research projects that can be considered ‘filler’. These projects unlock either very niche or very common place items on the station; a choice between unlocking magboots, hydroponic trays, or buildable research disk terminals is not compelling for the player. 
+-	A high proportion of 'filler' tech at lower research tiers leads to players considering such technologies merely as stepping stones towards some final project (typically a specific high tier technology). This also means that useful low tier technologies that belong to disciplines outside of their target are more likely to be overlooked during the early to mid game stages of the shift when they would be most useful.
+- Due to the abundance of low to mid-tier researachable items available on the station at round start, there is a limited need for other departments to request equipment from the science department
+-	Many research items seem to be under costed with respect to the current ease of points generation. This leads to shifts where all research projects can be readily completed, particularly during longer than normal shifts and / or shifts that have a large research team.
```

Comment:
in my opinion research __should__ be completed before the end of an abnormally long shift, thats because to do a project you not only have to have researched it but then make it and some projects will require things from multiple tech trees which takes further time.

Reply:
also a ccvar for a research cost multiplier might be an easy solution to make research slower.

Reply:
there is a separate idea of https://github.com/space-wizards/docs/pull/329 but overall - nope.

I don't think flat multiplier going to do any good job, as costs are not just 'flat low', they are not even balanced agains each other very much. multipler will just make it feel worse. It looks like it needs careful rebalance :(

Reply:
I mean. Both is good. Yes, all the research costs have to be rebalanced, but a cvar to adjust all the final values by a multiplier is would be a good inclusion as well